jgrapht-core/src/main/java/org/jgrapht/graph/AsWeightedGraph.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2007-2018, by Lucas J Scharenbroich and Contributors.
+ * (C) Copyright 2018, by Lukas Harzenetter and Contributors.
```

Comment:
Better to use `2018-2018` since we run a release script which assumes that format.